site/content/internal/infrastructure/kubernetes/kubernetes-troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Kubernetes Troubleshooting"
+date: 2018-11-07T15:24:42+01:00
+weight: 10
+---
+
+This page helps developers access and perform any type of maintenance in the Production Mattermost Kubernetes Cluster, which is running on AWS using EKS.
+
+## Set up a local environment to access Kubernetes (K8s)
+
+1. Make sure you have `kubectl` version 1.10 or later installed. If not, follow [these instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+2. Use your OneLogin account to retrieve AWS Keys for the main Mattermost AWS account following [these instructions](../../onelogin-aws)
+
+NOTE: When using Onelogin-aws account for Kubernetes configuration, please select the main Mattermost AWS account.
+
+### Configure kubectl for Amazon EKS
+
+Follow [these instructions](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html)
```

Comment:
Is this section part of setting up a local environment to access K8s?

Reply:
yes